utils/issue-format-bot/index.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 AJ Jordan
+//
+// This file is part of the HTTPS Everywhere issue management bot.
+//
+// The HTTPS Everywhere issue management bot is free software: you can
+// redistribute it and/or modify it under the terms of the GNU Affero
+// General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// The HTTPS Everywhere issue management bot is distributed in the
+// hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE.  See the GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with the HTTPS Everywhere issue management bot.  If
+// not, see <http://www.gnu.org/licenses/>.
```

Comment:
OK. I put these in there since this is what [FSF recommends](https://www.gnu.org/licenses/gpl-howto.html), but they're removed in 7b2dd77857c0c5a3c1bd63c7235721341a0b7a44 anyway :thumbsup: